e2e_test/e2e_test.go

```diff
@@ -460,6 +503,84 @@ var _ = g.Describe("E2E integration testing", g.Ordered, func() {
 				}, timeout, interval).Should(BeTrue())
 			})
 		})
+
+		g.Context("With more ServiceFunctionChains than resources", g.Ordered, func() {
+			g.It("Should have initially not SFCs", func() {
+				sfcLs := testutils.SfcList(dpuSideClient, vars.Namespace)
+				Expect(sfcLs.Items).To(BeEmpty())
+			})
+
+			var sfcNum int
+
+			g.It("Should initially detect number of DPU resources", func() {
+				dpuResNum := getMaxDpuResources(dpuSideClient)
+				Expect(dpuResNum).To(BeNumerically(">=", 2))
+				sfcNum = (dpuResNum / 2) + 1
```

Comment:
It would be easier to read if sfcNum = dpuResNum/2, and the loop below just loops until sfcNum followed by a check if all of them are running, followed by creating 1 more sfc which will be pendingf.